### PR TITLE
SDI-265: ♻️  Minor improvements

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: server/middleware/authorisationMiddleware.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import asyncMiddleware from './asyncMiddleware'
 
 export default function authorisationMiddleware(authorisedRoles: string[] = []): RequestHandler {
   return asyncMiddleware((req, res, next) => {
-    if (res.locals && res.locals.user && res.locals.user.token) {
+    if (res.locals?.user?.token) {
```
Comment (petergphillips) [inline server/middleware/authorisationMiddleware.ts:9]: intellij optional chaining suggestion

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -37,12 +38,5 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
...
 
-  njkEnv.addFilter('initialiseName', (fullName: string) => {
-    // this check is for the authError page
-    if (!fullName) {
-      return null
-    }
-    const array = fullName.split(' ')
-    return `${array[0][0]}. ${array.reverse()[0]}`
-  })
+  njkEnv.addFilter('initialiseName', initialiseName)
```
Comment (petergphillips) [inline server/utils/nunjucksSetup.ts:41]: extracted out to utils function so can test

File: server/utils/utils.test.ts [hunk 3]
```diff
@@ -1,31 +1,30 @@
-import convertToTitleCase from './utils'
+import { convertToTitleCase, initialiseName } from './utils'
 
-describe('Convert to title case', () => {
-  it('null string', () => {
-    expect(convertToTitleCase(null)).toEqual('')
+describe('convert to title case', () => {
+  it.each([
+    [null, null, ''],
```
Comment (petergphillips) [inline server/utils/utils.test.ts:5]: simplify test by passing in the test data

File: server/views/partials/layout.njk [hunk 4]
```diff
@@ -1,5 +1,4 @@
 {% extends "govuk/template.njk" %}
-{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
```
Comment (petergphillips) [inline server/views/partials/layout.njk:2]: unused

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**